frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_appearance_view.dart

```diff
@@ -19,8 +24,15 @@ class SettingsAppearanceView extends StatelessWidget {
           return Column(
             crossAxisAlignment: CrossAxisAlignment.start,
             children: [
-              ThemeModeSetting(currentThemeMode: state.themeMode),
-              ThemeSetting(currentTheme: state.appTheme.themeName),
+              ThemeModeSetting(
```

Comment:
Can we change the ordering here like this:
1. Theme
2. Theme Mode
3. Font Family
The logic behind it is that we want the theme and theme mode closer.
I don't know honestly, does it make sense?